pkg/openstack/loadbalancer.go

```diff
@@ -474,6 +475,21 @@ func getBoolFromServiceAnnotation(service *corev1.Service, annotationKey string,
 	return defaultSetting
 }
 
+// getProxyProtocolFromServiceAnnotation searches a given v1.Service the ServiceAnnotationLoadBalancerProxyEnabled to guess if the proxyProtocol needs to be
+// enabled and return the ProxyProtocol's version which is need to be applied
+func getProxyProtocolFromServiceAnnotation(service *corev1.Service) (bool, *v2pools.Protocol) {
```

Comment:
this could return just `*v2pools.Protocol`? boolean is not really needed.

Reply:
in all other places where `svcConf.enableProxyProtocol` is used, you could just use `svcConf.proxyProtocolVersion != nil`